src/index.js

```diff
@@ -0,0 +1,204 @@
+import './styles/styles.css';
+
+const refresh = document.querySelector('.uil-sync');
+const input = document.querySelector('input');
+const todosParent = document.querySelector('.todos-container');
+const clearCompletedTaskBtn = document.querySelector('button');
+
+// Object
+class MyObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const myArray = [];
```

Comment:
- [ ] As requested by the [previous reviewer](https://github.com/ecekpo/to_do_list_project/pull/1#discussion_r943673640) and according to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), you are required to  add some example tasks to populate your project.